### ISSUE TITLE
Some leaf types become invisible and/or look strange after a certain distance from the player

### ISSUE THREAD

**Issue by Levelowel** [IrisShaders/Iris#411]
Steps to reproduce:
1) use the latest commit of Iris 1.17 with no other mods installed
2) high fov & high render distance (110 and 24 in this screenshot) to make the bug more visible
3) don't use shaders, otherwise it won't appear
4) look at birch/spruce/acacia and probably other leaves that are far away

Log:
[latest.log](https://github.com/IrisShaders/Iris/files/6682514/latest.log)

F3 Screenshot:
![2021-06-20_16 32 11](https://user-images.githubusercontent.com/53264315/122678016-2f86a000-d1e5-11eb-8d76-66f38728210e.png)

**Comment by satriamjati** [comment]
110 and 24 works fine for me

**Comment by x60id** [comment]
Unfinished rendering but enough to notice distance.
![2021-06-21_04 34 11](https://user-images.githubusercontent.com/33513323/122689868-0b13dd80-d250-11eb-871c-96faea1cbd73.png)
[latest.log](https://github.com/IrisShaders/Iris/files/6683167/latest.log)
It seems alright using Iris Standalone 1.17.

**Comment by coderbot16** [comment]
This almost certainly has to do with mipmapping, and I'm like 90% sure what causes it.

**Comment by Levelowel** [comment]
Yes, I tested it again with different mipmapping levels and setting it to off or 1 seems to fix the problem.

**Comment by AViewFromTheTop** [comment]
This now happens in the latest developmental 1.18 version of Iris that was put under Releases, where Iris and Sodium are separated. The versions where Sodium was bundled with Iris didn't have this issue, and I believe that manually building an earlier version of 1.18 Iris without Sodium also didn't have this issue.

**Comment by Track07-cda** [comment]
It might relate to the graphic driver settings. I have the same issue when running on the NVIDIA graphic card, but no such issue when using integrated Intel graphics.

**Comment by coderbot16** [comment]
The development versions had this bug resurface for a bit, but now it is only observable without Sodium installed, which is now only possible when running in a development environment.

**Comment by coderbot16** [comment]
The fix for this bug (only currently observable in a dev env) would likely be to add a core shader that fixed up the alpha test as needed for our bettermipmaps patches.
